crates/api/src/op_store.rs

```diff
@@ -112,13 +119,13 @@ pub trait OpStore: 'static + Send + Sync + std::fmt::Debug {
     ///
     /// - As many op ids as can be returned within the `limit_bytes` limit.
     /// - The total size of the op data that is pointed to by the returned op ids.
-    /// - A new timestamp to be used for the next query..
+    /// - A new timestamp to be used for the next query.
     fn retrieve_op_ids_bounded(
         &self,
         arc: DhtArc,
         start: Timestamp,
-        limit_bytes: usize,
-    ) -> BoxFuture<'_, K2Result<(Vec<OpId>, usize, Timestamp)>>;
+        limit_bytes: u32,
```

Comment:
The limit config is u32 and I can't see us needing > u32::MAX bytes in one go :)